react/components/form/FieldTemplate.tsx

```diff
@@ -4,7 +4,7 @@ import { FieldTemplateProps } from 'react-jsonschema-form'
 import { ComponentEditorFormContext } from '../EditorContainer/Sidebar/typings'
 
 interface Props {
-  children?: React.ReactElement<any>
+  children?: React.ReactElement<unknown>
```

Comment:
Isn't just `React.ReactElement` fine here?

Reply:
I think `React.ReactNode` would be more accurate; I'll change it.